features/contain-layout.yml

```diff
@@ -0,0 +1,6 @@
+name: Layout containment
+description: "The `contain: layout` CSS declaration TODO."
```

Comment:
```suggestion
description: "The `contain: layout` CSS declaration isolates the inside of an element such that it cannot affect the layout of the rest of the page or be affected by the rest of the page. This permits the browser to avoid slower layout calculations."
```